packages/vat-data/src/index.js

```diff
@@ -1,2 +1,7 @@
 export * from './vat-data-bindings.js';
 export * from './kind-utils.js';
+
+// Reexporting from here to minimize breakage during transition.
+// Everyone should get it directly from store rather than vat-data.
+// TODO Remove once it won't break anything.
+export { objectMap } from '@agoric/store';
```

Comment:
It looks like this PR has already moved all references. When I check out this branch and do "Find all references" on this export, there are none.

I think it would be worth removing it now so autocomplete gets it from the right place.

Removing it is a breaking change to the package API so for semver it should make it into CHANGELOG.md as a breaking change. (That would require a `BREAKING CHANGE` string or `!` modifier in the commit message. ) But this package hasn't reached 1.0 yet so all bets are off and I'd be very surprised if any code outside SDK imports it.

Reply:
Done